### ISSUE TITLE
✅ 🐞 Bug Report: Popup's new `Accessible Navigation` option inadvertently auto-selects the first navigable element inside Popups - [ED-9869]

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#21409]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi @rami-elementor👋

After updating Elementor Pro to version 3.11, an annoying pop-up problem occurs. The problem is that if we have an input in the popup like a search widget, the keyboard will be activated automatically if the **mobile users** open the popup.
This is a simple but very annoying problem for users (website visitors).

**It should be noted that I experienced this bug with Android phone - Chrome browser.**

### Steps to reproduce

1- Create a popup for a mobile dropdown menu.
2- Put a search widget or any other input inside it.
3- View the website on mobile and open the popup. After opening the pop-up, the keyboard will be activated immediately.

**If you roll back Elementor Pro to version 3.10.2, this issue is resolved.**

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

[system-info-localhost-20-02-2023.txt](https://github.com/elementor/elementor/files/10787217/system-info-localhost-20-02-2023.txt)

**Comment by tobias992** [comment]
I can confirm. I use icon-list-widget and the first item have the "active" border. Not nice.

Edit:
Disable "Accessible navigation" in popup settings works fine for me.

**Comment by 79ho3ein** [comment]
> I can confirm. I use icon-list-widget and the first item have the "active" border. Not nice.
> 
> Edit: Disable "Accessible navigation" in popup settings works fine for me.

O yeah. You are right. Did this option exist before? I have never seen it before. So now this is not a bug and I will close this issue.

Thanks for the information🥂

**Comment by nicholaszein** [comment]
Hello @79ho3ein! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by 79ho3ein** [comment]
Hi @nicholaszein. 

This is not a bug. That's why I closed this thread. If we turn off the Accessible navigation option, this problem will be solved.
Strange, but I've never seen this option before. But there it is anyway, and it fixes the problem.
The only problem now seems to be that this option is on by default for pop-ups. But it will be much better if it is disabled by deafult.

**Comment by averyjane98** [comment]
I am having the same issue. But when I turn of accessible navigation the pop-up menu either wont open at all or will appear as an entirely white box.

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.17.0! 🥳 

The **`Accessible Navigation`** option in Popups exists to assist users in navigating the content inside the Popup. It's a crucial feature for a more accessible website.

Previously, users experienced a navigation challenge where the focus, upon triggering a Popup, would land on a random first focusable element like a button or a link. This often led to confusion, especially for users relying on assistive technologies, as the focus could be on an element located at the end of the Popup.

To enhance user experience, we've implemented a crucial change. Now, when a modal opens, the focus immediately shifts to the close button. This adjustment ensures a more intuitive and accessible interaction, allowing users, particularly those with disabilities, to easily locate and use the close button if they wish to exit the Popup. 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂

**Comment by nicholaszein** [comment]
Hi @averyjane98.

The issue you're reporting sounds a lot more like a problem with cache. I would advise you to clear all levels of cache applicable to your site.

If you still experience this issue, then please submit a new bug report and follow all the instructions in the form so we can take at look at your case.

Kind regards.
